lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -300,6 +310,37 @@ class QcFlagService {
         });
     }
 
+    /**
+     * Create verification of QC flag
+     * @param {Partial<QcFlagVerification>} parameters flag verification parameters
+     * @param {object} relations QC Flag entity relations
+     * @param {UserWithRoles} relations.userWithRoles user with roles
+     * @return {Promise<QcFlag>} promise
+     * @throws {NotFoundError|AccessDeniedError}
+     */
+    async verifyFlag(parameters, relations) {
+        return dataSource.transaction(async () => {
+            const { flagId, comment } = parameters;
+
+            const qcFlag = await this.getOneOrFail(flagId);
+
+            const { userWithRoles: { userId, externalUserId } } = relations;
```

Comment:
Why expecting roles if you don't use it?

Reply:
removed